src/app/query/stacks/balance/crypto-asset-balances.utils.ts

```diff
@@ -106,7 +106,7 @@ export function addQueriedMetadataToInitializedStacksFungibleTokenAssetBalance(
     balance: createMoney(
       assetBalance.balance.amount,
       metadata.symbol ?? '',
-      metadata.decimals ?? undefined
+      metadata.decimals ?? 0
     ),
```

Comment:
Not for this PR, but we should try and think of a way never to need to initialise a blank, non-existent Money instance. Needing to do this highlights design problems elsewhere in the codebase.